src/libs/ComponentSpecs.js

```diff
@@ -400,25 +379,25 @@ const prototypeSpecs = {
     }
   },
 
-  /**
-   * remove all click listeners and debug dashboards
-   * @method _removeSubComponentsListenersAndDebugDashboards
-   * @private
-   */
-  _removeSubComponentsListenersAndDebugDashboards() {
-    const self = this;
-    if (self._subI13nComponents && self._subI13nComponents.length > 0) {
-      self._subI13nComponents.forEach((subI13nComponent) => {
-        subI13nComponent.componentClickListener.remove();
```

Comment:
fun fact, this should be `unsubscribe` instead of `remove`
https://github.com/yahoo/subscribe-ui-event